### DISCUSSION TITLE
Install metamask extension while running test cases

### DISCUSSION THREAD

**Discussion by shivam-ln** [cypress-io/cypress#24553] [Component Testing]
Hi my web application is written in Next.JS. I want to write test cases for that application but issue is that  for some most of the functionality i need metamask extension need to be installed and need to interact with metamask and do transactions . 
is it possible to use metamask ?

**Comment by stavares843** [comment]
https://github.com/saxenashivang/cypress-metamask-v2
https://medium.com/coinmonks/test-e2e-login-to-dapp-with-metamask-with-synpress-5248dd1f17c1

in case that's helpful 🔨
